lib/sycamore/sycamore/materialize.py

```diff
@@ -50,7 +49,7 @@ def __init__(
         child: Optional[Node],
         context: Context,
         path: Optional[Union[Path, str, dict]] = None,
-        source_mode: MaterializeSourceMode = MaterializeSourceMode.OFF,
+        source_mode: MaterializeSourceMode = MaterializeSourceMode.RECOMPUTE,
```

Comment:
I think the docstrings documenting the different `source_mode` options in `docset.py` should be updated, yes? It would not hurt to document the different settings in the docstring for this class as well while you're at it.

Reply:
I updated that when I added the new ones. I found this as a place that I'd missed updating.  Since I'd potentially missed others I did the grep and have now fixed those as well. I fixed them to the new canonical sycamore.MATERIALIZE_RECOMPUTE and sycamore.MATERIALIZE_USE_STORED constants
I added a docstring pointer from the enum class to the DocSet materialize method. I'm not a fan of duplicate documentation (I find it gets out of date), but if you want it I can add it.